### DISCUSSION TITLE
Android 13 Input Controll is Restricted by Android  due to "Security reasons"

### DISCUSSION THREAD

**Discussion by encrystudio** [rustdesk/rustdesk#6241] [General]
### Bug Description

I can't use the Input Controll feature because Android blocks the Accessibility setting of RustDesk due to security reasons. Is there a workaround? 

### How to Reproduce

Try to enable Input Controll on Android 13 Settings

### Expected Behavior

I expected that I can enable the Accessibility Service of RustDesk

### Operating system(s) on local side and remote side

Android 13 (MUI14) 

### RustDesk Version(s) on local side and remote side

1.2.3

### Screenshots

![Screenshot_2023-10-31-09-50-59-675_com miui videoplayer](https://github.com/rustdesk/rustdesk/assets/94799008/1a19b46c-ccb2-4702-aebd-c4717dece984)

**Comment by djordans** [comment]
https://www.youtube.com/watch?v=0bcLjpfrmHw

**Reply by encrystudio** [reply]
Thanks, worked

**Reply by wasd52030** [reply]
@djordans tkanks,worked

**Reply by nacesprin** [reply]
Is there any way to remember the Input control permission and not being acepting manually each time I need to share screen?

**Reply by djordans** [reply]
Yes with adb

adb shell appops set com.carriez.flutter_hbb PROJECT_MEDIA allow

**Comment by eslamgpt8** [comment]
![Screenshot_٢٠٢٥-٠٥-٢٠-٢٣-٥٣-٥٤-٤٨٧_com android settings-edit](https://github.com/user-attachments/assets/cc1ac4fc-0c25-4b41-8b4c-c9ef9f9eed76)


I have this problem. Sorry for taking a screenshot in Arabic, but the message says service is malfunctioning.
